WeaponMechanics/src/main/java/me/deecaad/weaponmechanics/weapon/stats/WeaponStat.java

```diff
@@ -0,0 +1,85 @@
+package me.deecaad.weaponmechanics.weapon.stats;
+
+import java.util.Arrays;
+
+public enum WeaponStat {
+
+    UUID("VARCHAR(255) NOT NULL", String.class),
+    WEAPON_TITLE("VARCHAR(255) NOT NULL", String.class),
+
+    SKIN("VARCHAR(255)", String.class),
+    OFF_SKIN("VARCHAR(255)", String.class),
```

Comment:
`OFF_SKIN` implies a different skin while the weapon is in the off hand...
`HAND_SKIN` might be a better name since it is the actual skin of the hand